### ISSUE TITLE
Update v2.x branch to be in maintenance mode

### ISSUE THREAD

**Issue by pragnagopa** [Azure/azure-functions-host#6972]
Related to V2 to V3 auto-upgrade

V2 version:  https://github.com/Azure/azure-functions-host/releases/tag/v2.0.14786 is the last version before we started auto-upgrade

v2.x branch should now be maintenance mode. Only critical bug fixes that need a release need to be merged into v2.x

**Comment by surgupta-msft** [comment]
@pragnagopa is this issue still relevant? Let me know if any action item is pending on this.
